### ISSUE TITLE
Invalid path in submodule ...

### ISSUE THREAD

**Issue by jesko42** [dreamworksanimation/openmoonray#15]
Submodule path 'moonray/mcrt_denoise': checked out '710ca0bf75552b2f81e4a79f5ab614173a4c468c'
Submodule path 'moonray/moonray': checked out 'ba155b14779586254212c6973421ac2e7fcb47e9'
Submodule path 'moonray/moonray_arras/mcrt_computation': checked out '2e778a66ae685c7df11d61278f6f159cb9efab65'
Submodule path 'moonray/moonray_arras/mcrt_dataio': checked out '6c84f98dc6341d6d414228bc573c8d2b5d745240'
Submodule path 'moonray/moonray_arras/mcrt_messages': checked out '3e050e36e3db25433d27d7da75e34c5715e302d6'
error: invalid path 'houdini/otls/Vop::DW_MOONRAY::AttributeMap::1.hda'
Submodule path 'moonray/moonray_gui': checked out '606b411880cecab5a38392ee46a2edad7e652a7f'
Submodule path 'moonray/moonshine': checked out 'faa2940107c04676d4dd2537a8c60b62a2ec504b'
Submodule path 'moonray/moonshine_usd': checked out 'efe09ee4b75d678005061d121cf832fc3f2d9a9a'
Submodule path 'moonray/render_profile_viewer': checked out '44bb5d66aa5295cc1176823d04a12cc4cd009e76'
Submodule path 'moonray/scene_rdl2': checked out 'fb0c969026b1e5a3c16cc75ee3ec9ff7df85ad98'
Unable to checkout 'ce861215cf2c4fec77c8decdabd96755917ae742' in submodule path 'moonray/moonray_dcc_plugins'

**Comment by jixingcn** [comment]
Yes, me too.

**Comment by xh5a5n6k6** [comment]
It looks like the same problem in #8, and it works for me to do the same thing described there :).

**Comment by jixingcn** [comment]
> It looks like the same problem in #8, and it works for me to do the same thing described there :).

You are right. The file path is different between Windows and Linux. The project is only for Linux right now.
